lib/Config.ts

```diff
@@ -147,7 +157,14 @@ class Config {
     this.lnd.BTC = {
       disable: false,
       certpath: path.join(lndDefaultDatadir, 'tls.cert'),
-      macaroonpath: path.join(lndDefaultDatadir, 'data', 'chain', 'bitcoin', this.network, 'admin.macaroon'),
+      macaroonpath: path.join(
```

Comment:
This sort of style change is less significant as it doesn't break consistency, but as @erkarl said it makes the PR harder to review and I'd prefer to limit code churn and keep any changes in this PR relevant to its purpose.

Reply:
Totally, these were made before I disabled Prettier for this workspace... I will revert them